continuous-testing/use-cases/markmap.md

```diff
@@ -1,56 +1,115 @@
 * Continuous Testing
     * For Development APIs
-        * Unit Testing
-            * xUnit (e.g. Junit\[Java\], PyUnit\[Python\], NUnit\[C#\], CppUnit\[C++\]
-            * Tools
-                * Java
-                * Python
-                * C#
-                * C/C++
-        * Integration, Dependency and Stub Mocking
-            * Simulate dependencies
-            * Stub out coupled services for planned results
-            * Mock specific application behavior
-            * Tools
-                * Java
-                    * Mockito
-                    * EasyMock
-                    * JMock
-                * Python
-                    * 
-                * C#
-                * C/C++
-        * Code Quality Analysis
-            * Semmle
+        * For module testing (xUnit)
+            * Using (Java) [JUnit](https://junit.org/junit5/) [&#x1F3C1; STARTER KIT(TBD)](../starter-kits/#unit-teting-with-java)
```

Comment:
I remember you mentioning the flag emoji (🏁) was not displayable in the editor you were using, and you used the HTML `&#x1F3C1;` instead. The checkered flag is part of Unicode 6.0 (from 2010), so editors from that year and beyond should support this (as would most modern browsers). 

Two questions:
- Should we be supporting editors that are not implementing modern Unicode? Or should we ask people to update their editors? The reason I'm asking is that Markdown is supposed to be _readable_, unlike markup for HTML. Embedding coded arguments like this go against that principle. 
- How does one translate a Unicode flag like 🏁 to `&#x1F3C1;`? We should probably document this in the future within our contributing guide.

Reply:
Good points. In the markdown you could use `:checkered_flag:` to get the character, but then it wouldn't display right in the html rendering. So the entity code makes it work with both. When rendered it displays right.

I have a Mac version of UltraEdit which is feature parity with a 2016 revision of UltraEdit (v21.x). In 2016 it didn't support Unicode 6.0, and was integrated in v24.x in 2017. Lots of programs have glitches and it takes years to get around to fully implementing it. Even [Apple's TextEdit had problems](https://support.apple.com/guide/textedit/if-characters-arent-displayed-correctly-txted1028/mac). BBEdit or Notepad++ might work but I don't regularly use those. Even Git may need encoding settings updated. 

[This website](https://www.compart.com/en/unicode/) allows name search of Unicode characters. [Search for "flag" returns a set of these characters.](https://www.compart.com/en/unicode/search?q=flag#characters)

Reply:
I guess this is just a risk, maybe not the end of world. Anything broken can be fixed when an earlier version is in Git.